client/package.json

```diff
@@ -29,6 +29,9 @@
     ]
   },
   "dependencies": {
+    "@dnd-kit/core": "^6.0.8",
```

Comment:
The three dnd-kit packages are remnants of my first attempt at dnd, and are not needed. The react-beautiful-dnd package is all you need.